### PR TITLE
Fix indentation problem after multi-line Python function definition

### PR DESCRIPTION
**Pull request by FoSuCloud**
## References

Fixes https://github.com/jupyterlab/jupyterlab/issues/14668

## Code changes

Use resolutions to limit the @codemirror/lang-python version
image

Execute "yarn install" and "jlpm build" to update dependencies
image

## User-facing changes

nothing
## Backwards-incompatible changes

nothing

### CODE REVIEW

**Comment by FoSuCloud** [comment]
@meeseeksdev tag bug

File: package.json [hunk 1]
```diff
@@ -106,6 +106,7 @@
     "watch:packages": "python scripts/watch_packages.py"
   },
   "resolutions": {
+    "@codemirror/lang-python": "^6.1.3",
```
Comment (fcollonval) [inline package.json:109]: We should avoid using `resolutions` as much as possible. In this case, could you upgrade the minimal dependency version in [jupyterlab/packages/codemirror/package.json](https://github.com/jupyterlab/jupyterlab/blob/f699d32d19e1687bf27e4c83318f50062adb0c5a/packages/codemirror/package.json#L51) to `^6.1.3`.

```suggestion
```

> For information, if you want to upgrade a third-party dependency in the lock file without changing any package.json, you should use the yarn command [yarn up](https://yarnpkg.com/cli/up).

**Review by fcollonval — CHANGES_REQUESTED** [review]
thanks @FoSuCloud 

Could you update the specification a bit differently (see below comment)?

> GitHub tip: if you use [special keywords](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) like _Fixes_ in front of the issue in the PR description, the issue will be linked with this PR easing documentation and repository management :wink:

File: package.json (hunk 1 above)
Comment (FoSuCloud) [reply package.json:109]: ok, i'll fix it tonight

**Review by krassowski — APPROVED** [review]
Thank you @FoSuCloud

**Comment by krassowski** [comment]
@meeseeksdev please backport to 4.0.x

**Pull request merged**